### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by izdeby**
Updated README.me of the repo.

### CODE REVIEW

**Comment by izdeby** [comment]
@cpuhrsch

File: README.md [hunk 1]
```diff
@@ -1,5 +1,110 @@
-#  Under construction
+PLEASE NOTE, NESTEDTENSORS ARE IN AN ACTIVE DEVELOPMENT AND EVERYTHING HERE IS A SUBJECT TO CHANGE.
```
Comment (cpuhrsch) [inline README.md]: *under active development

**Comment by cpuhrsch** [comment]
Can you also delete the construction sign asset?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,110 @@
...
+## Constraints
+ - Each Tensor constituent of the list it represents, if any, must be of its dtype, layout and device. 
+ - The dimension of a constituent Tensor must be one less than the dimension of the NestedTensor. 
+ - An empty list of Tensors yields a NestedTensor of dimension one. 
+ - Each constituent NestedTensor must be of its dtype, layout and device. 
+ - The dimension of a constituent NestedTensor must be one less than the dimension of the NestedTensor.
+
+## Prerequisites
+
+You will need this packages to be installed:
```
Comment (cpuhrsch) [inline README.md]: *these packages

File: README.md [hunk 3]
```diff
@@ -1,5 +1,110 @@
...
+- pytorch
+- torchvision
```
Comment (cpuhrsch) [inline README.md]: torchvision is not necessary for nestedtensor itself

File: README.md [hunk 4]
```diff
@@ -1,5 +1,110 @@
...
+## Build 
+Run 
+```
+python setup.py develop
+```
+
+## Usage
+Import nested tensors and torch via ```from nestedtensor import torch```
+### Creation
+You can create a nester tensor by initializing its values direcrtly or by creating tensors first and then add them to nested tensor.
```
Comment (cpuhrsch) [inline README.md]: *directly

File: README.md [hunk 5]
```diff
@@ -1,5 +1,110 @@
...
+```
+@tensorwise()
+@tensorwise()
+def simple_fn(t1, t2):
+    return t1 + 1 + t2
+
+
+a = torch.tensor([1, 2])
+b = torch.tensor([7, 8])
+print(simple_fn(a, b))
```
Comment (cpuhrsch) [inline README.md]: Can you also include what this prints and a nestedtensor example?

File: README.md [hunk 6]
```diff
@@ -1,5 +1,110 @@
...
+For all other ops, please, see examples.
+
+
+## The tensorwise decorator
+The nestedtensor package allows the user to decorate existing functions with a tensorwise decorator. This decorator lifts the given function to check for NestedTensor arguments and apply recursively apply it to their constiuents with all other arguments untouched.
+
+Decorating the function as tensorwise does not affect its behavior with respect to non-NestedTensor arguments. In particular, the tensorwise decorator will search all arguments for a NestedTensor and if none is found dispatch to exactly the given function.
+
+```
+@tensorwise()
```
Comment (cpuhrsch) [inline README.md]: I think you meant to only use this once

File: README.md [hunk 7]
```diff
@@ -1,5 +1,110 @@
...
+    return t1 + 1 + t2
+
+
+a = torch.tensor([1, 2])
+b = torch.tensor([7, 8])
+print(simple_fn(a, b))
+```
+
+## Contribution
+The project is in an active development. If you have a suggestions or have found an issue, please file an issue!
```
Comment (cpuhrsch) [inline README.md]: *under active developemnt

File: README.md [hunk 8]
```diff
@@ -1,5 +1,111 @@
...
+@tensorwise()
+def simple_fn(t1, t2):
+    return t1 + 1 + t2
+
+
+a = torch.tensor([1, 2])
+b = torch.tensor([7, 8])
+print(simple_fn(a, b))
+
+>> tensor([ 9, 11])
```
Comment (cpuhrsch) [inline README.md]: Still no example using a NestedTensor

**Pull request merged**